Utils/Dataflow/pyDKB/dataflow/stage/tests/test_ProcessorStage.py

```diff
@@ -55,81 +55,74 @@ class ProcessorStageArgsTestCase(unittest.TestCase):
 
     def setUp(self):
         self.stage = pyDKB.dataflow.stage.ProcessorStage()
+        self.args = dict(self.default_args)
```

Comment:
Maybe name it `expected_args` (or `exp_args`)? Longer, right, but more readable... it\`s just now it makes me remid myself every now and then how it is used.

Reply:
Done.